features/admin.connections.v1/pages/connection-edit.tsx

```diff
@@ -445,7 +445,7 @@ const ConnectionEditPage: FunctionComponent<ConnectionEditPagePropsInterface> =
          * @param cb - Callback.
          * @returns Promise containing authenticator details.
          */
-        const getAuthenticatorDetails = <T extends unknown>(cb: (id: string) => Promise<T>) => {
+        const getAuthenticatorDetails = <T,>(cb: (id: string) => Promise<T>) => {
```

Comment:
is this comma intentional?

Reply:
Yes, this is the convention with the upgraded eslint version. Suggested by quick-fix option.